tests/Feature/Base/SharingTestBase.php

```diff
@@ -226,16 +226,4 @@ protected function generateExpectedAlbumJson(string $albumID, string $albumTitle
 			'parent_id' => $parentAlbumID,
 		], $attrToMerge);
 	}
-
-	protected function generateExpectedSmartAlbumJson(
-		bool $isPublic,
-		?string $thumbID = null,
-		array $expectedPhotos = []
-	): array {
-		return [
-			'is_public' => $isPublic,
-			'thumb' => $this->generateExpectedThumbJson($thumbID),
-			'photos' => $expectedPhotos,
-		];
-	}
```

Comment:
Why have you moved this method?

Reply:
re-added now that the policies are always present.